client/src/app/site/motions/services/motion-csv-export.service.ts

```diff
@@ -51,6 +102,11 @@ export class MotionCsvExportService {
                     label: 'state',
                     map: motion => this.motionRepo.getExtendedStateLabel(motion)
                 };
+            } else if (option === 'text') {
+                return {
+                    label: 'Final text',
```

Comment:
I would prefere to use just 'Text' because it could be also the original version - which is not the final text.